Essentials/src/main/resources/messages.properties

```diff
@@ -853,42 +1178,82 @@ totalWorthAll=\u00a7aSold all items and blocks for a total worth of \u00a7c{1}\u
 totalWorthBlocks=\u00a7aSold all blocks for a total worth of \u00a7c{1}\u00a7a.
 tpCommandDescription=Teleport to a player.
 tpCommandUsage=/<command> <player> [otherplayer]
+tpCommandUsage1=/<command> <player>
+tpCommandUsage1Description=Teleports you to the specified player
+tpCommandUsage2=/<command> <player> <other player>
+tpCommandUsage2Description=Teleports the first specified player to the second
 tpaCommandDescription=Request to teleport to the specified player.
 tpaCommandUsage=/<command> <player>
+tpaCommandUsage1=/<command> <player>
+tpaCommandUsage1Description=Requests to teleport to the specified player
 tpaallCommandDescription=Requests all players online to teleport to you.
 tpaallCommandUsage=/<command> <player>
+tpaallCommandUsage1=/<command> <player>
+tpaallCommandUsage1Description=Requests for all players to teleport to you
 tpacancelCommandDescription=Cancel all outstanding teleport requests. Specify [player] to cancel requests with them.
 tpacancelCommandUsage=/<command> [player]
+tpacancelCommandUsage1=/<command>
+tpacancelCommandUsage1Description=Cancels all your outstanding teleport requests
+tpacancelCommandUsage2=/<command> <player>
+tpacancelCommandUsage2Description=Cancels all your outstanding teleport request with the specified player
 tpacceptCommandDescription=Accepts a teleport request.
 tpacceptCommandUsage=/<command> [otherplayer]
+tpacceptCommandUsage1=/<command>
+tpacceptCommandUsage1Description=Accepts the last incoming teleport request
 tpahereCommandDescription=Request that the specified player teleport to you.
 tpahereCommandUsage=/<command> <player>
+tpahereCommandUsage1=/<command> <player>
+tpahereCommandUsage1Description=Requests for the specified player to teleport to you
 tpallCommandDescription=Teleport all online players to another player.
-tpallCommandUsage=/<command> <player>
+tpallCommandUsage=/<command> [player]
+tpallCommandUsage1=/<command> [player]
+tpallCommandUsage1Description=Teleports all players to you, or another player if specified
 tpautoCommandDescription=Automatically accept teleportation requests.
-tpautoCommandUsage=/<command> <player>
+tpautoCommandUsage=/<command> [player]
+tpautoCommandUsage1=/<command> [player]
+tpautoCommandUsage1Description=Toggles if tpa requests are auto accepted for yourself or another player if specified
 tpdenyCommandDescription=Reject a teleport request.
 tpdenyCommandUsage=/<command>
+tpdenyCommandUsage1=/<command>
+tpdenyCommandUsage1Description=Rejects an incoming teleport request
```

Comment:
`/tpaccept` usage mentions the last teleport request specifically, but it's not mentioned here (?)

Reply:
altered tpaccept